lib/utils/docker.coffee

```diff
@@ -418,3 +428,13 @@ copyQemu = (context) ->
 	.then ->
 		fs.chmod(binPath, '755')
 	.return(binPath)
+
+# if required, change the permissions (mode) of files added to the tar stream
+# This is needed for windows as the execute bit isn't supported there
+chmodTarEntry = (mode) ->
+	os = require('os')
+	if os.platform() == 'win32'
+		mode |= 0o111
+		mode &= 0o755
+	else
+		mode
```

Comment:
I'm not too familiar with coffee script and this was the only way I could find to return `mode` unchanged on non-windows platforms. If there is a more idiomatic way of doing it, I'll change it.

Reply:
There's a variety of other ways to do this, but this is fine imo, don't worry about it (we're in the process of moving this to TypeScript anyway, so the exact details of this kind of thing don't matter too much)